docs/PARAMETER.md

```diff
@@ -30,6 +30,28 @@ When a variable is removed from the configuration file and the file is pushed, i
 
 When pulling configuration, use `--store-command=refs` flag to restore the configuration as Parameter Store references that can be added to the secrets section of a Task Definition.
 
+### Using cStore CLI with AWS Parameter Store
+
+To pull your configuration using the `cstore pull` CLI command, you must include the following **two** statements in any relevant resource policies:
```

Comment:
`To make the parameters accessible using the AWS API, Account Roles, or the CLI command `$ cstore pull`, use these resource policy statements.` Removing word like `your` and `you` make the writing more abbreviated, specific, and technical in nature.